src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
-* Use the **Experience** section and footer in DevTools **Performance** panel to dive deep and debug against specific Core Web Vitals.
-* Use **Search Console's Core Web Vitals report** for a summary of how your origins are performing in the field.
-* Use the **Web Vitals Extension** to track a page's performance against Core Web Vitals in real-time.
+<figure class="w-figure">
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/LqEnonUKd8j3QGK3Ax7w.png", alt="A collection of icons for Google tools. From left to right, the icons represent 'CrUX on BigQuery', 'CrUX API', 'PSI API', 'web-vitals.js', with 'Lighthouse CI' at the far right.", width="800", height="185" %}
+</figure>
+
+The last step in the performance improvement workflow cycle is two-fold:
 
-We will cover more about our Core Web Vitals tooling at [web.dev Live](/live/) in June. Sign up to get updates on the event!
+1. Use [CrUX on BigQuery](/chrome-ux-report-bigquery/), [CrUX API](/chrome-ux-report-api/), [PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started), and/or the [`web-vitals`](https://www.npmjs.com/package/web-vitals) JavaScript library to automate a website's field data collection, and optionally use this data to power custom dashboards and alerting systems.
+2. Use [Lighthouse-CI](https://github.com/GoogleChrome/lighthouse-ci) for performance regression testing.
+
+{% Aside %}
+[Research by Google](https://youtu.be/YJGCZCaIZkQ?t=112) has shown that most performance improvements tend to regress within six months. A website needs continuous monitoring in both the lab _and_ the field to identify worsening trends in Core Web Vitals and other performance metrics to avoid regressions.
+{% endAside %}
 
-~ by Elizabeth and Addy, WebPerf Janitors
+The tools listed in the first step may seem like a lot, but you don't necessarily need _all_ of them&mdash;only those that will give you the level of coverage you need to assess performance and preserve good user experiences. The table below can help you determine which tools you need:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>CrUX on BigQuery</th>
+        <th>CrUX API</th>
+        <th>PSI API</th>
+        <th>web-vitals.js</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Origin-level data (Field only)</td>
+        <td>URL and Origin-level data (Field only)</td>
+        <td>URL and Origin-level data (Lab and Field)</td>
+        <td>URL-level data (Field only)</td>
+      </tr>
+      <tr>
+        <td>Monthly aggregated data</td>
+        <td>28 days rolling</td>
+        <td>28 days rolling</td>
+        <td>Real-time</td>
+      </tr>
+      <tr>
+        <td>Allows slicing of data in meaningful ways which can be joined with other public datasets like the <a href="https://httparchive.org/">HTTP Archive</a> for advanced insights.</td>
```

Comment:
The fact that this can be joined with HTTP Archive data is useful for anyone doing large-scale analysis of the web, but I'm not sure if it's useful for an individual site owner trying to improve performance.

Probably the most useful part of the CrUX BigQuery dataset to site owners is the breakdowns by country, connection, device, etc. as well as the more detailed distribution data.